tools/stimulus/types-RC.yaml

```
@@ -564,10 +564,15 @@ EDGE_INDICES:
     HEADER: {}
 
 EXTRA:
-    CALL: '0'
+    CALL: ~
```

We can leave this unchanged now.

```suggestion
    CALL: '0'
```